### ISSUE TITLE
View Boundaries in Continuous View

### ISSUE THREAD

**Issue by Lenbus** [musescore/MuseScore#20811]
### Issue type

UI bug (incorrect info or interface appearance)

### Bug description

When I select Continuous View (btw. one of my favorite features <3) the upper bound of the page is most often cropped, so that text, chord symbols or rehearsal marks are not displayed within the white area of the (continuous) page and just seem to vanish in the background. 

### Steps to reproduce

Open a score with multiple staffs (like e.g. choir music) and switch to Continuous View.

### Screenshots/Screen recordings

![grafik](https://github.com/musescore/MuseScore/assets/12497326/211bcd0b-a7cc-4679-9ca7-2f2a6ee1e80e)

### MuseScore Version

4.2

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

Windows 11

**Comment by MarcSabatella** [comment]
That shouldn't happen - the view should include enough space to accommodate whatever the "music top margin" setting is (Format / Style / Page), and the default value should allow quite a bit of content above the top staff without causing problems.  Looks like maybe your score has turned this value down.  So for now, you should consider resetting that to the default, or even increasing further if necessary.

But that said, it certainly would be better if space were allocated automatically so you didn't have to worry about that setting.

If your score is at the default and still has this problem, please ZIP and attach it here.  but the picture certainly looks like you've turned it down to next to nothing.

**Comment by Lenbus** [comment]
Thanks for your reply, @MarcSabatella. 
Oh, I guess you've found the cause: I have set the upper margin to 3 pt. Changing that back to e.g. 8 pt resolves the issue, but that is not, what I would consider to be a good bet, as Continuous view ist just for the working process and not to be presented in any way. Moreover upper margin of 2 or 3 pt ist just, what best suits my need for tablet layout of my scores and looks absolutely fine in "Page View". 

Continuous:
P1: ![grafik](https://github.com/musescore/MuseScore/assets/12497326/d673c385-f508-4cda-9195-92356292dd3d)
P2: ![grafik](https://github.com/musescore/MuseScore/assets/12497326/1d990a8d-a811-4a0c-ab21-321348304eed)

Page (all looking fine):
P1: ![grafik](https://github.com/musescore/MuseScore/assets/12497326/38bb8dc6-52a0-401b-99eb-ff4473110e50)
P2: ![grafik](https://github.com/musescore/MuseScore/assets/12497326/c9f32574-4b63-40ec-9883-bed4fba97280)

Regards. :)

**Comment by MarcSabatella** [comment]
Indeed, fiddling with the music top margin is just the workaround.  Ideally MuseScore would add more space as needed automatically in continuous view just as it already does in page view.

So bottom line, to me it's still a bug.  A tricky one, though, since calculating the appropriate of *extra* space to leave above and beyond the music top margin requires analyzing the full score, and that's something we try to avoid doing very often since it slows things down.
In MU3, the extra text would be pushed off the screen also at first, but you could then drag the canvas down to see more.  In MU4, you can drag the canvas down, but the extra text spills onto the background where it's harder to see - or impossible if you've changed the background to black as you apparently have

Perhaps a quick fix could be, force the background to the same as the paper color when in continuous view.  Not sure that distinction between paper and background serves any useful purpose in continuous view.

**Comment by bkunda** [comment]
See this video for what I think is the real issue here:

https://github.com/musescore/MuseScore/assets/86290556/255a26e8-6f2e-42a7-9a53-4f9b3d7fef00

I think it'd make most sense if the MusicTopMargin and MusicBottomMargin simply had no effect in continuous view.
